### PR TITLE
TECH-6905 modernize dive sailthru client

### PR DESCRIPTION
**Pull request by wbrown22**
[TECH-6905 modernize dive sailthru client](https://industrydive.atlassian.net/browse/TECH-6905)

### Non-technical Context
we want to modernize the dive sailthru client so that it is compatible with both python 2 and 3

### Technical Context
ran modernize on this repo and made the suggested changes. Then I ran the tests in python 2 and python 3 to make sure it passes both ways.

### Manual Testing Instructions
Checkout this branch and run the following commands:
* make -> so that you get the new dependency
* python setup.py test -> python2 test running
* python3 setup.py test -> python3 test running
 
#### Developer

- [x] Ensure code complies with the [style guide](https://industrydive.atlassian.net/wiki/spaces/TECH/pages/639500407/Tech+Team+s+Style+Guides)
- [ ] Devise edge cases to test the bug fix or feature being merged
- [ ] Make sure the code is clean and understandable
- [ ] Ensure code is not overly inefficient
- [ ] Ensure documentation is understandable and accurate, including:
    - [ ] Code comments and doc strings
    - [ ] Technical documentation
- [ ] Set deployment notes on Jira ticket

#### Reviewer

- [ ] Ensure code complies with the [style guide](https://industrydive.atlassian.net/wiki/spaces/TECH/pages/639500407/Tech+Team+s+Style+Guides)
    - If there are few or minor issues, add inline GitHub comments
    - If there are many issues or it seems like the developer did not follow the style guide, leave a comment asking them to do so
- [ ] Devise edge cases to test the bug fix or feature being merged
- [ ] Make sure the code is clean and understandable
    - [ ] Ask for inline comments on unclear sections
- [ ] Ensure code is not overly inefficient
- [ ] Ensure documentation is understandable and accurate, including:
    - [ ] Code comments and doc strings
    - [ ] Technical documentation
- [ ] Read any deployment notes on the Jira ticket and ensure they can be followed

### CODE REVIEW

**Comment by MiriamSexton** [comment]
I'm supposed to run `make pthonqa` or `make test`? Because the first one gives me an error

File: README.rst [hunk 1]
```diff
@@ -4,3 +4,16 @@ Dive Sailthru Client
...
 This is intended to be a drop-in replacement for the official client at <https://github.com/sailthru/sailthru-python-client>. That means it preserves all the original API functions (though some are improved with e.g. better error detection) and adds some new features, namely better exception handling for all API functions plus some new functions to conveniently request campaign metadata annotated with a guess on the type of email it is and the dive publication (misnamed as key `dive_brand`) it refers to.
+
+Alternative for running tests
+-----------------------------
+If you have issues using the make command for running tests,
+first set the environment variables in the command line with the following command:
+
+export SAILTHRU_API_KEY="place key for dev account here" .   
+
+export SAILTHRU_API_SECRET="place secret for dev account here"
```
Comment (MiriamSexton) [inline README.rst]: 👍
